x-pack/filebeat/input/azureblobstorage/job.go

```diff
@@ -29,16 +31,29 @@ import (
 const jobErrString = "job with jobId %s encountered an error: %w"
 
 type job struct {
-	client       *blob.Client
-	blob         *azcontainer.BlobItem
-	blobURL      string
-	hash         string
-	offset       int64
+	// azure blob client handle
```

Comment:
The convention for field comments reflects the convention for exported label comments; `// client is an azure blob handle.`